### ISSUE TITLE
About the pre-trained of Convnext !

### ISSUE THREAD

**Issue by baowenshuwh** [PaddlePaddle/PaddleDetection#7494]
### 问题确认 Search before asking

- [X] 我已经搜索过问题，但是没有找到解答。I have searched the question and found no related answer.

### 请提出你的问题 Please ask your question

 I found that you have convnext as a pre-training weight for the yolox backbone network, can I port it to ultralytics/yolov5? The training epoch here is 36, why not 300?

**Comment by nemonameless** [comment]
Training more epochs should have higher accuracy. 
36e is just an example of achieving high accuracy in a short time.

**Comment by baowenshuwh** [comment]
Ok, I see. Thank you for your answer
